### DISCUSSION TITLE
deltaLake, Iceberg, old-fashioned hive in one catalog?

### DISCUSSION THREAD

**Discussion by tooptoop4** [trinodb/trino#13528] [General]
can I create a single catalog ie 'hive' and query some deltaLake tables, some iceberg tables, some normal hive/parquet tables? or do I need to create 3 separate catalogs?

**Comment by findepi** [comment]
You currently need to create 3 catalogs and configure table redirections between them.

**Reply by tooptoop4** [reply]
rats! was hoping users did not need to worry about that implementation detail since its all the same hive metastore and they don't need to know if something is view vs table vs materialised view.
